### PR TITLE
Game Instance - Database

### PR DESCRIPTION
**Pull request by AscendedWise**
- Finished the Database view
- Added a sendData method to database that sends an Arraylist<String> copy of the database to the leaderboardView

### CODE REVIEW

File: src/main/java/Database/LeaderBoard.java [hunk 1]
```diff
@@ -21,7 +21,7 @@ protected Player(String name, int wins){
             this.name = name;
             this.wins = wins;
         }
-        // Good coding practice bla bla getter functions
+        // Good coding practice blah blah getter functions
```
Comment (123Chan) [inline src/main/java/Database/LeaderBoard.java:24]: lol

**Review by 123Chan — APPROVED** [review]

**Comment by 123Chan** [comment]
Awesome work on the database system, I tried it and it works. I see the draws counts as a win as well. The button for the name entry could say "Enter" or something, but it's really insignificant. Great job, I approved.

File: src/main/java/Database/LeaderBoard.java (hunk 1 above)
Comment (nawarani) [inline src/main/java/Database/LeaderBoard.java:24]: Who did this

Comment (123Chan) [reply src/main/java/Database/LeaderBoard.java:24]: lol

**Review by nawarani — APPROVED** [review]
Looks good, ready for merge. Great work on integrating this!

**Pull request merged**